job-service-db/src/main/resources/db/migration/V8__add_correlation_id_to_job_tables.sql

```diff
@@ -0,0 +1,27 @@
+--
+-- Copyright 2016-2025 Open Text.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+/*
+ **************************************************************
+ ******************** Update tables ***************************
+ **************************************************************
+ */
+
+ALTER TABLE job
+    ADD COLUMN IF NOT EXISTS correlation_id VARCHAR(48) NULL;
+
+ALTER TABLE job_task_data
+    ADD COLUMN IF NOT EXISTS correlation_id VARCHAR(48) NULL;
```

Comment:
We need to ensure the upgrade instructions take account of these changes.

See https://gitlab.otxlab.net/cybersecurity-enterprise/cddri/packaging/application-install-package/-/tree/main/upgrade-procedures/24.4.2.HF1?ref_type=heads for previous example.

Reply:
I have added this to the comments of the ticket